contracts/pm/mixins/MixinWrappers.sol

```diff
@@ -19,6 +21,8 @@ contract MixinWrappers is MTicketBrokerCore {
         uint256[] _recipientRands
     )
         public
+        currentRoundInitialized
```

Comment:
Let's run the `whenSystemNotPaused()` check first - this is the ordering we use in the BondingManager and also skips the round initialized check when the system is actually paused.

Reply:
fixed in 752bec9